lib/global_uid/base.rb

```diff
@@ -52,12 +42,11 @@ def self.drop_uid_tables(id_table_name)
     end
 
     def self.init_server_info
-      id_servers = self.global_uid_servers
-      increment_by = self.global_uid_options[:increment_by]
-      connection_timeout = self.global_uid_options[:connection_timeout]
+      id_servers = GlobalUid.configuration.id_servers
+      increment_by = GlobalUid.configuration.increment_by
+      connection_timeout = GlobalUid.configuration.connection_timeout
 
       raise "You haven't configured any id servers" if id_servers.nil? or id_servers.empty?
-      raise "More servers configured than increment_by: #{id_servers.size} > #{increment_by} -- this will create duplicate IDs." if id_servers.size > increment_by
```

Comment:
This was a nice static check to have.. Any reason for the removal?

Reply:
The check is still there, it now lives in the configuration class and happens earlier. The client will receive an exception during configuration, instead of during execution.